platform/contracts/admin/src/msg.rs

```diff
@@ -48,13 +46,13 @@ pub enum SudoMsg {
     ChangeDexAdmin { new_dex_admin: Addr },
     RegisterProtocol { name: String, protocol: Protocol },
     MigrateContracts(MigrateContracts),
+    ClearStorage {},
```

Comment:
please add a doc string explaining the purpose of that its intended for internal usage